tests/identity/test_registry.py

```diff
@@ -60,6 +60,7 @@ def test_get_provider_error(monkeypatch: MonkeyPatch) -> None:
 
 
 def test_get_provider() -> None:
+    SETTINGS_STORE.reset()
```

Comment:
hm isn't this handled by the `isolate_settings` fixture?

Reply:
Nope, the `settings` fixture loads the `BaseSettings` before `DummySettings` is loaded. Hence, `DummySettings.get()` will cause an Error. So resetting is necessary prior to loading `DummySettings`.